lib/messager.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+require 'date'
+
+class Messager
+
+  def send_text
+    account_sid = "ACc640aebc9ed667fe38f3481bab876368"
+    auth_token = "134e02cb4127679d53aaed26bd40af7d"
+    client = Twilio::REST::Client.new account_sid, auth_token
+
+    from = "+441702806346" # Your Twilio number
+
+    friends = {
+      "+4407767831356" => "Valentina"
+    }
+    friends.each do |key, value|
+      client.account.messages.create(
+        :from => from,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)